bouncerscript/task.py

```diff
@@ -85,3 +89,19 @@ def check_product_names_match_aliases(context):
 
     if not all(validations):
         raise TaskVerificationError("The product/alias pairs are corrupt: {}".format(aliases))
+
+
+def check_locations_match(locations, product_config):
+    # TO IMPROVE: in the future, if we wanted to improve this even more, we could
+    # start processing the XML and compare bitwise the locations returned from
+    # bouncer
+    return len(locations) == len(product_config)
```

Comment:
`len()` is not a great test, but it's something. Sounds like this is tricky due to xml; can you expand on this?
If we parsed the xml into a python dict or something, would that help in sanity checking?

Reply:
I'll try to parse XML. That's the correct and real check anyways. I'm sure the XML dom can get me something useful. Otherwise, I'll parse or alike.